pysat/instruments/pysat_sgp4.py

```diff
@@ -820,37 +949,43 @@ def add_msis(inst, glat_label='glat', glong_label='glong',
         msis['Nn_O'] = pt.nn['O']
         msis['Nn_O2'] = pt.nn['O2']
         msis['Tn_msis'] = pt.Tn_msis
-        msis_params.append(msis)        
+        msis_params.append(msis)
     # print 'Complete.'
     msis = pds.DataFrame(msis_params)
     msis.index = inst.data.index
     inst[msis.keys()] = msis
-    
+
     # metadata
-    inst.meta['Nn'] = {'units':'cm^-3',
-                       'desc':'Total neutral number particle density from MSIS.'}
-    inst.meta['Nn_N'] = {'units':'cm^-3',
-                         'desc':'Total nitrogen number particle density from MSIS.'}
-    inst.meta['Nn_N2'] = {'units':'cm^-3',
-                          'desc':'Total N2 number particle density from MSIS.'}
-    inst.meta['Nn_O'] = {'units':'cm^-3',
-                         'desc':'Total oxygen number particle density from MSIS.'}
-    inst.meta['Nn_O2'] = {'units':'cm^-3',
-                          'desc':'Total O2 number particle density from MSIS.'}
-    inst.meta['Tn_msis'] = {'units':'K',
-                            'desc':'Neutral temperature from MSIS.'}
+    inst.meta['Nn'] = {'units': 'cm^-3',
+                       'desc': 'Total neutral number particle density ' +
```

Comment:
Is it ok to use the + operator here? I think in one of my recent reviews of standards I saw that + and strings only works in a particular python interpreter.

Reply:
Depending on the code, I've been using a mix.  I'd prefer to use `''.join` everywhere, but that can be tricky.  We can discuss at the next meeting.

Reply:
Another thing, is that we should determine whether it's more pysatic to use 'units' or inst.meta.units_label when setting the meta data